readthedocs/settings/docker_compose.py

```diff
@@ -18,10 +18,14 @@ class DockerBaseSettings(CommunityDevSettings):
     PRODUCTION_DOMAIN = 'community.dev.readthedocs.io'
     PUBLIC_DOMAIN = 'community.dev.readthedocs.io'
     PUBLIC_API_URL = f'http://{PRODUCTION_DOMAIN}'
+
     SLUMBER_API_HOST = 'http://web:8000'
+    SLUMBER_USERNAME = 'admin'
+    SLUMBER_PASSWORD = 'admin'
```

Comment:
These look to be new, are these settings intentional?

Reply:
These are required for builds to hit the API. Otherwise, we get 403 and builds don't work.

This problem is not new, but I suppose that you may have these settings overwritten in a `local_settings.py` file?

They are required in docker environment to be able to build docs.

Reply:
We can remove them from this PR and figure it out in another one, tho.

Reply:
Yea, seems odd this would be related to this PR. It must have been working before somehow?

Reply:
It's completely unrelated to this PR, yes. I will remove it from it and add it to another one so we can discuss it there. We will see if it fails for you as well when building.